openedx/core/djangoapps/content_libraries/tests/test_runtime.py

```diff
@@ -17,14 +15,14 @@
 from openedx.core.djangoapps.xblock import api as xblock_api
 from openedx.core.lib import blockstore_api
 from student.tests.factories import UserFactory
+from xmodule.unit_block import UnitBlock
 
 
 class UserStateTestBlock(XBlock):
     """
     Block for testing variously scoped XBlock fields.
     """
     BLOCK_TYPE = "user-state-test"
-    has_score = False
```

Comment:
Turns out my test case was catching this, but I "fixed" it for the tests by declaring it on this test XBlock instead of realizing it needed to be added to all blocks that don't declare it.